Resources/Audio/_CD/Voice/Avali/attributions.yml

```diff
@@ -0,0 +1,9 @@
+- files: ["avali_scream"]
+  license: "CC-BY-SA-3.0"
+  copyright: "Taken from Steam mod / Starbound"
+  source: "https://steamcommunity.com/sharedfiles/filedetails/?id=3001955034"
+
+- files: ["avali_laugh"]
+  license: "CC-BY-SA-3.0"
+  copyright: "Original owner unknown - Edited by Ridiue (discord id: 295382088226439189)"
+  source: "https://drive.google.com/file/d/12hE8CZsNNY_hmm3ewHhnYB-Dd0_4x7x5/view"
```

Comment:
You can't just take something from google drive or a steam mod and give it a license.

Reply:
You need to figure out what licence this file is licensed under and change the credits here (or finding another clip if the license is not comparable). That may require contacting the author.